src/config/modelProviders/groq.ts

```diff
@@ -3,6 +3,27 @@ import { ModelProviderCard } from '@/types/llm';
 // ref https://console.groq.com/docs/models
 const Groq: ModelProviderCard = {
   chatModels: [
+    {
+      displayName: 'LLaMA3.1 405B',
+      enabled: true,
+      functionCall: true,
+      id: 'llama-3.1-405b-reasoning',
+      tokens: 131_072,
+    },
+    {
+      displayName: 'LLaMA3.1 70B',
+      enabled: true,
+      functionCall: true,
+      id: 'llama-3.1-70b-versatile',
+      tokens: 131_072,
+    },
+    {
+      displayName: 'LLaMA3.1 8B',
+      enabled: true,
+      functionCall: true,
+      id: 'llama-3.1-8b-instant',
+      tokens: 131_072,
+    },
     {
       displayName: 'LLaMA3 70B',
       enabled: true,
```

Comment:
If Llama 3.1 70B is enabled, then Llama 3 70B can be disabled by default, and this line can be deleted